EventFilter/CTPPSRawToDigi/python/ctppsDigiToRaw_cfi.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+from EventFilter.RawDataCollector.rawDataCollector_cfi import *
+
+#from EventFilter.CTPPSRawToDigi.ctppsPixelRawData_cfi import ctppsPixelRawData
+#ctppsPixelRawData.InputLabel = cms.InputTag("RPixDetDigitizer")
+
+ctppsPixelRawData = cms.EDProducer("CTPPSPixelDigiToRaw",
+    InputLabel = cms.InputTag("RPixDetDigitizer"),
+    mappingLabel = cms.string("RPix") 
+)
+
+ctppsTotemRawData = cms.EDProducer("CTPPSTotemDigiToRaw",
+    InputLabel = cms.InputTag("RPSiDetDigitizer")
+)
+
+ctppsRawData = cms.Sequence(ctppsTotemRawData*ctppsPixelRawData)
```

Comment:
sequences should not be defined in cfi files. Please move to cff.
Normally, a cfi file should define only one module instance with its default parameters.

For this file this means:
- ctppsPixelRawData_cfi and ctppsTotemRawData_cfi are needed . Note, that if the fillDescriptions methods are defined appropriately, these files will be generated by the framework
- this file can be renamed to ctppsRawData_cff or ctppsDigiToRaw_cff